frontend/src/components/templates/ILab/ILabFilters.jsx

```diff
@@ -0,0 +1,173 @@
+import "react-date-picker/dist/DatePicker.css";
+import "react-calendar/dist/Calendar.css";
+
+import {
+  Chip,
+  ChipGroup,
+  Switch,
+  Toolbar,
+  ToolbarContent,
+  ToolbarItem,
+} from "@patternfly/react-core";
+import {
+  setIlabCatFilters,
+  setIlabSubCatFilters,
+  setIlabTypeFilter,
+} from "@/actions/ilabActions.js";
+import { useDispatch, useSelector } from "react-redux";
+
+import DatePicker from "react-date-picker";
+import { FilterIcon } from "@patternfly/react-icons";
+import PropTypes from "prop-types";
+import SelectBox from "@/components/molecules/SelectBox";
+import { formatDate } from "@/utils/helper";
+import { removeIlabAppliedFilters } from "@/actions/ilabActions";
+import { setDateFilter } from "@/actions/filterActions.js";
+import { useNavigate } from "react-router-dom";
+
+export const ILabFilters = (props) => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const {
+    onSwitchChange,
+    isSwitchChecked,
+    start_date,
+    end_date,
+    navigation,
+    type,
+    filterData,
+  } = props;
+  const {
+    subCategoryOptions,
+    categoryFilterValue,
+    subCategoryFilterValue,
+    typeFilterOptions,
+    typeFilterValue,
+    appliedFilters,
+  } = useSelector((state) => state.ilab);
+  const startDateChangeHandler = (date, key) => {
+    setDateFilter(date, key, navigation, type);
+  };
+  const endDateChangeHandler = (date, key) => {
+    setDateFilter(date, key, navigation, type);
+  };
+  const onCategoryChange = (_event, value) => {
+    dispatch(setIlabCatFilters(value));
+  };
+  const onSubCategoryChange = (_event, value) => {
+    dispatch(setIlabSubCatFilters(value));
+  };
+  const onTypeFilterChange = (_event, value) => {
+    dispatch(setIlabTypeFilter(value, navigate));
+  };
+  const deleteItem = (key, value) => {
+    dispatch(removeIlabAppliedFilters(key, value, navigation));
+  };
+  return (
+    <>
+      {filterData.length > 0 && (
+        <Toolbar
+          id="filter-toolbar"
+          ouiaId="data_table_filter"
+          clearAllFilters={() => console.log("hey")}
```

Comment:
"hey"?? 😁 I suspect there's little if any value to putting debug "bread crumbs" like this into the repo!